lib/ibm/cloud/sdk_http/connection.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require_relative 'sdk_response'
+
+module IBM
+  module Cloud
+    module SDKHTTP
+      # Class to wrap the HTTP Provider implementation.
+      class HTTPConnection
+        include HTTParty
+
+        def initialize(logger)
+          self.class.logger(logger)
```

Comment:
initialize doesn't return it's value, so this line is a no-op...unless it has side effects?  If so, can it just be called later?

Reply:
Typically there is a `logger=` method...or does HTTPPart not have that?